lte/cloud/go/services/lte/obsidian/handlers/handlers.go

```diff
@@ -300,21 +300,35 @@ func listEnodebs(c echo.Context) error {
 	if nerr != nil {
 		return nerr
 	}
+	pageSize, pageToken, err := obsidian.GetPageSizeAndTokenParams(c)
```

Comment:
Nit: functions with the word "and" in them are clunky. Try `GetPaginationParams`